main.js

```diff
@@ -23,12 +23,15 @@ function displayNumbers(allNumbers) {
         number.setAttribute('class', 'numberBox');
         document.querySelector('#play_area').appendChild(number);
     }
-
 }
 
 document.querySelector('#start').addEventListener('click', () => {
+    const randomNumbers = generateNumbers();
+
```

Comment:
why empty line?

Reply:
We separated defining a variable and the function so it's easier to read.